test/users/npadmana/blas/test_blas3.chpl

```diff
@@ -0,0 +1,519 @@
+use Random;
```

Comment:
OK, that's perfectly reasonable - I was just curious if they were derived from somewhere else really.